infra/Pos/Slotting/Impl/Ntp.hs

```diff
@@ -168,11 +168,12 @@ ntpGetCurrentSlot var = ntpGetCurrentSlotImpl var >>= \case
         Nothing <$ printSlottingData
   where
     printSlottingData = do
-        sd <- getSlottingData
+        eli <- getEpochLastIndex
+        sd  <- getEpochSlottingData eli
```

Comment:
There should be `getLastEpochSlottingData` which returns `EpochSlottingData` instead of `Maybe EpochSlottingData` and it should be user here.